algo-dev-demo/h2o-python-mojo/README.md

```diff
@@ -0,0 +1,92 @@
+# H2O MOJO Python Insurance Claims demo
+
+In this demo we'll go through deploying a pre-trained [H2O](https://www.h2o.ai) model for insurance claims predictions.
+This model is saved as a MOJO (Model Object, Optimized).
+For more information on these objects see the [H2O documentation](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/mojo-quickstart.html).
+
+This whole demo uses the [H2O python library](http://h2o-release.s3.amazonaws.com) to train and make prediction of the model.
+The model was trained using the `train.py` script located on this directory but this MOJO model
+could also be trained, saved and served using other H2O supported languages.
+
+This demo features the usage of an special environment designed to run H2O models in algorithmia.
+
+Other examples with H2O:
+- Deploying H2O models (Python)
+
+## Prerequisites
+
+Fork this repository so you have access to the algorithm code and data files. We will use the Algorithmia Web IDE in this demo but note that you can also use a repository [from Github](https://algorithmia.com/developers/algorithm-development/source-code-management) or you can use the [CLI](https://algorithmia.com/developers/clients/cli/) to deploy your model instead once you've [created an algorithm](https://algorithmia.com/developers/algorithm-development/languages/python/#create-an-algorithm) and cloned your repository.
+
+## Upload pre-trained model to Data Collections
+
+We are going to host the serialized pre-trained model on the Algorithmia platform, for more info look at the [Data Collections docs](https://algorithmia.com/developers/data/hosted).
+
+To create a Data Collection:
+
+- Log in to your Algorithmia account and click on **Data Sources** on the left navigation sidebar
+- Click on **New Data Source** on the top right corner, select **Hosted Data Collection** and name it `h2o_demo`
+- You can now select or drag and drop the pre-trained model: [data/DeepLearning_model_python_1611090304383_1.zip](https://github.com/algorithmiaio/sample-apps/raw/master/algo-dev-demo/h2o-python-mojo/data/DeepLearning_model_python_1611090304383_1.zip) to upload it to the Algorithmia Hosted Collection
+- Once the file it's uploaded you can copy the URI by clicking the triple dots to the right of the file name (e.g. `data://<USERNAME>/h2o_demo/DeepLearning_model_python_1611090304383_1.zip`)
```

Comment:
"Once the file **is** uploaded**,** you can"

Reply:
Done